deep_speech_2/datasets/chime/chime3_test.py

```diff
@@ -0,0 +1,143 @@
+        print("\nMD5 Chesksum %s ..." % filepath)
+        if not md5file(filepath) == md5sum:
+            raise RuntimeError("MD5 checksum failed.")
+    else:
+        print("File exists, skip downloading. (%s)" % filepath)
+    return filepath
+
+
+def unpack(filepath, target_dir):
+    """Unpack the file to the target_dir."""
+    print("Unpacking %s ..." % filepath)
+    if filepath.endswith('.zip'):
+        zip = zipfile.ZipFile(filepath, 'r')
+        zip.extractall(target_dir)
+        zip.close()
+    elif filepath.endswith('.tar') or filepath.endswith('.tar.gz'):
+        tar = zipfile.open(filepath)
+        tar.extractall(target_dir)
+        tar.close()
+    else:
+        raise ValueError("File format is not supported for unpacking.")
+
+
+def create_manifest(data_dir, prefix, manifest_path):
+    """Create a manifest json file summarizing the data set, with each line
+    containing the meta data (i.e. audio filepath, transcription text, audio
+    duration) of each audio file within the data set.
+    """
+    print("Creating manifest %s ..." % manifest_path)
+    transcript_filepath = os.path.join(
+        data_dir, "CHiME3/data/transcriptions/et05_" + prefix + ".trn_all")
+    transcript = {}
+    for line in open(transcript_filepath, 'r'):
+        pos = line.find(' ')
+        key = line[:pos]
+        text = line[pos + 1:].strip().lower().replace('.', '')
+        transcript[key] = text
+    json_lines = []
+    for subfolder, _, filelist in sorted(os.walk(data_dir)):
+        for filename in filelist:
+            if filename.endswith('CH1.wav') and subfolder.endswith(prefix):
+                key = filename[:filename.find('.')]
+                filepath = os.path.join(data_dir, subfolder, filename)
+                audio_data, samplerate = soundfile.read(filepath)
+                duration = float(len(audio_data)) / samplerate
+                json_lines.append(
+                    json.dumps({
+                        'audio_filepath': filepath,
+                        'duration': duration,
+                        'text': transcript[key]
+                    }))
+    with open(manifest_path, 'w') as out_file:
+        for line in json_lines:
+            out_file.write(line + '\n')
+
+
+def main():
+    if not os.path.exists(os.path.join(args.target_dir, "CHiME3")):
+        # download
+        filepath_simu = download(URL_SIMU, MD5_SIMU, args.target_dir,
+                                 FILENAME_SIMU)
+        filepath_real = download(URL_REAL, MD5_REAL, args.target_dir,
+                                 FILENAME_REAL)
+        filepath_text = download(URL_TEXT, MD5_TEXT, args.target_dir,
+                                 FILENAME_TEXT)
+        # unpack
+        unpack(filepath_simu, args.target_dir)
+        unpack(filepath_real, args.target_dir)
+        unpack(filepath_text, args.target_dir)
+        unpack(
+            os.path.join(args.target_dir, 'CHiME3_isolated_et05_real.zip'),
```

Comment:
Why the two file names here are hardcoded while others are set through variables?